petclinic.yaml

```diff
@@ -7,6 +7,7 @@ items:
     name: petclinic
     labels:
       app: petclinic
+      component: webapp
```

Comment:
You could use [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/)

Reply:
Done. Please validate values https://github.com/redhat-developer/devfile/pull/3/commits/e32d03b740200a20219373e0a80f6249ba4283aa